docs/dev/nightly_data_builds.rst

```diff
@@ -344,8 +255,8 @@ the ``gcloud storage cp`` command, which behaves very much like the Unix ``cp``
 
    gcloud storage cp gs://builds.catalyst.coop/<build ID>/pudl.sqlite ./
 
-If you wanted to download all of the build outputs (more than 10GB!) you could use ``cp
--r`` on the whole directory:
+If you want to download all of the build outputs (~20GB!) you can do a recursive copy of
+the whole directory hierarchy:
```

Comment:
Should we warn people (mostly catalyst folks) that doing this will incur some cloud costs?

Reply:
Will mention that.